ValheimPlus/Configurations/Sections/BuildingConfiguration.cs

```diff
@@ -5,7 +5,7 @@ public class BuildingConfiguration : ServerSyncConfig<BuildingConfiguration>
         public bool noInvalidPlacementRestriction { get; set; } = false;
         public bool noWeatherDamage { get; set; } = false;
         public float maximumPlacementDistance { get; internal set; } = 5;
-
+        public float pieceComfortRadius { get; internal set; } = 0;
```

Comment:
I wonder if this would be better placed in the `[Items]` section. What do you think?

Reply:
Also I think this configuration would be better as the default radius of "10" and users can modify it how they like rather than a percentage